### PR TITLE
Use placeholder for unsubscribe link and place it near the normal email footer content.

### LINKED ISSUE #13330: Make email report unsubscribe link look consistent with other report footer links
Currently the unsubscribe link looks like this:

unsub

Proposal: instead let's make it 
* re-use the style of the existing footer content (grey text, light blue links, small font), 
* and possibly move the sentence just below ` The leading open analytics platform that respects your privacy. `, with an empty line, and remove the separator line?

follows up https://github.com/matomo-org/matomo/pull/13214 https://github.com/matomo-org/matomo/issues/2197

### PR DESCRIPTION
**Pull request by diosmosis**
Fixes #13330

@sgiehl can you review?

Here's what it looks like:

image

w/ WhiteLabel enabled:

image

### CODE REVIEW

**Comment by diosmosis** [comment]
Fixed the issue making the unsubscribe link not underlined, here's what it looks like now:

image

**Comment by sgiehl** [comment]
@diosmosis  could you do a rebase?

**Comment by diosmosis** [comment]
@sgiehl Rebased.

File: plugins/CoreHome/templates/_htmlEmailFooter.twig [hunk 1]
```diff
@@ -1,16 +1,26 @@
 {% set fontStyle %}color:{{ themeStyles.colorText|e('html_attr') }};font-family:{{ themeStyles.fontFamilyBase|e('html_attr') }};{% endset %}
+c{% set styleParagraph = 'font-size:15px;line-height:24px;margin:0 0 16px;' %}
```
Comment (sgiehl) [inline plugins/CoreHome/templates/_htmlEmailFooter.twig]: there's a `c` in front of that line that should be removed again

**Review by sgiehl — CHANGES_REQUESTED** [review]
Besides the little comment, looks good and works (also with Whitelabel)

**Review by sgiehl — APPROVED** [review]

**Pull request merged**